src/tasks/package.js

```diff
@@ -74,7 +74,7 @@ module.exports = (config) => {
         clean: 'del-cli dist',
         commitlint: 'commitlint',
         commitmsg: 'commitlint -e $GIT_PARAMS',
-        lint: 'eslint --cache src test',
+        lint: 'eslint --cache src test --ignore-pattern fixtures',
```

Comment:
`tasks/eslint` and add it to the `eslintignore` file instead please

Reply:
@michael-ciniawsky No, it is broke `eslint` ignoring (`eslint` uses `.gitignore` by default) and it and this will force us to have two files (`.gitignore` and `.eslintignore`) with a lot of the same content, it is bad idea. Please study tools and their api before request changes.

Reply:
`eslintignore` is currently added to each repo, but if we decide to remove it then agreed. Please not x via `eslintignore` and y via scripts